### PR TITLE
PIM-5210: Use the enhanced Product Edit Form in the mass edit common attributes action

### PR DESCRIPTION
**Pull request by grena**
| Q                 | A
| ----------------- | ---
| Specs             | Y
| Behats            | Y
| Blue CI           |
| Changelog updated | Y
| Review and 2 GTM  |
| Micro Demo (PO)   | Y
| Migration script  | N
| Tech Doc          | N

### CODE REVIEW

File: src/Pim/Bundle/EnrichBundle/Resources/public/spec/product/form/mass-edit/attributesSpec.js [hunk 1]
```diff
@@ -0,0 +1,83 @@
+/* global describe, it, expect, spyOn, beforeEach */
```
Comment (pierallard) [inline src/Pim/Bundle/EnrichBundle/Resources/public/spec/product/form/mass-edit/attributesSpec.js]: Weird to have this file in /src, no ?

**Comment by juliensnz** [comment]
GTM dude :)

File: src/Pim/Bundle/EnrichBundle/Resources/public/spec/product/form/mass-edit/attributesSpec.js (hunk 1 above)
Comment (grena) [inline src/Pim/Bundle/EnrichBundle/Resources/public/spec/product/form/mass-edit/attributesSpec.js]: All our Jasmine files are in src on 1.4.
Maybe a topic that can be raised on master to put them in a dedicated directory

**Comment by skeleton** [comment]
GTM !!

**Pull request merged**